README.md

```diff
@@ -44,12 +44,25 @@ We welcome contributors! After you get your developoment environment setup, revi
 
 ## Setting up Development
 
-1. Use the same GitHub fork you created when you "Set Up Live App" or create a new one: Click Fork > Create New Fork
-2. On your newly created fork page, click the green Code button and copy the URL to the clipboard
-3. On your machine (e.g. these instructions are for Mac) run the usual `git clone [url you copied]`
-4. `cd ` into that directory and `cat .tool-versions` to see what version of ruby the project is using. Check your local ruby version (`ruby --version`) and install the version the project is using.
-5. `psql --version` to ensure you have postgres installed and running. Any version 16.x should work.
-6. `bundle install`
-7. `bin/rails db:setup`  < Note: This will load the sample fixture data into your database
-8. `bin/rails dev`  < Starts up all the services
-9. Point your browser to `localhost:3000` and register as a new user
+The easiest way to get up and running is to use the provided docker compose workflow:
+
+1. Make sure you have [Docker Desktop](https://docs.docker.com/desktop/) installed and running.
+2. Clone your fork `git clone [repository url]`
+3. `cd` into your clone.
+4. Run `docker-compose up --build` to start the app.
+5. Open [http://localhost:3000](http://localhost:3000) and register as a new user.
```

Comment:
Let's add some instructions about rails console, db console, and rails tests. Let's add to the README how to launch each of these. Or, actually, maybe there's a more general guidance which gets you into a docker console where I can execute any arbitrary command like `bundle install` when I am upgrading a gem. I'm fuzzy on how to do that.

I'm pretty sure tests is `docker-compose run test` but when I try that I get an error about the `app_test` database not being initialized. Do we need an extra conditional in the docker-entrypoint to detect and initialize this test database?

Reply:
Good idea. In the past I've added scripts to the bin folder to do tests and psql and such, but I didn't do that here. 

As a result, the commands are all a little verbose -- `docker compose run server rails test`. Folks can use shell history or their own aliases still, but maybe it's worth just having a bin script for it?

Reply:
I don't have a strong opinion, I think I'd have to use it more to decide so I'd say just run w/ whatever you think.

But is it a typo that you said `run server rails test` ? Shouldn't it be `run rails test` ?